core/src/main/java/io/undertow/server/protocol/http2/Http2UpgradeHandler.java

```diff
@@ -76,26 +76,8 @@ public void handleRequest(HttpServerExchange exchange) throws Exception {
         final String settings = exchange.getRequestHeaders().getFirst("HTTP2-Settings");
         if(settings != null && upgrade != null
                 && upgradeStrings.contains(upgrade)) {
-            if(HttpContinue.requiresContinueResponse(exchange) && false) {
```

Comment:
Please skip this change. View this discussion for background info: https://github.com/undertow-io/undertow/commit/55445de19844484f02e169dbbc6d9b0754d926c8

Reply:
I created UNDERTOW-2135 to properly handle this: https://issues.redhat.com/browse/UNDERTOW-2135

Reply:
I reverted the change and added //NOSONAR so that Sonar won't report on it.  I also added the URL for the issue.

Reply:
UNDERTOW-2135 is already solved, so there is no need for this